ios/Podfile.lock

```diff
@@ -310,15 +312,18 @@ PODS:
     - React-jsi (= 0.63.4)
   - RNCAsyncStorage (1.13.2):
     - React
+  - RNGestureHandler (1.9.0):
+    - React-Core
   - RNKeychain (6.2.0):
     - React
-  - RNSVG (12.1.0):
+  - RNSVG (9.13.6):
```

Comment:
Don't really care, but just curious. Any ideas why this change happened? Looks like the new `react-native-linear-gradient` dependency triggered a downgrade on this native module `RNSVG`, which is probably ok for now. Just wanted to hear from you if you have any deeper knowledge regarding this than me. But if not, no problem at all.

Reply:
reverted this change  by removing linear gradient